automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/cluster/ClusterMonitoringSpec.scala

```diff
@@ -1,21 +0,0 @@
-package org.broadinstitute.dsde.workbench.leonardo.cluster
-
-import org.broadinstitute.dsde.workbench.fixture.BillingFixtures
-import org.broadinstitute.dsde.workbench.leonardo.LeonardoTestUtils
-import org.scalatest.{FreeSpec, ParallelTestExecution}
-
-class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with ParallelTestExecution with BillingFixtures {
-
-  "Leonardo clusters" - {
-
-    "should save cluster error if the userscript fails" in {
```

Comment:
It made no sense to have this spec with a single case. Functionality moved to `ClusterStatusTransitionsSpec`.